File: backend/HDP.Domain/Models/Endereco.cs
using System;
using System.Collections.Generic;

namespace HDP.Persistence
{
    public partial class Endereco
    {
        public int EnderecoId { get; set; }
        public string Logradouro { get; set; } = null!;
        public int Numero { get; set; }
        public string Cidade { get; set; } = null!;
        public string Estado { get; set; } = null!;
        public int TutorId { get; set; }

        public virtual Tutor Tutor { get; set; } = null!;
    }
}

File: backend-adonis/app/Models/Endereco.ts
import { DateTime } from 'luxon';
import { BaseModel, column } from '@ioc:Adonis/Lucid/Orm';

export default class Endereco extends BaseModel {
  @column({ isPrimary: true })
  public id: number;

  @column()
  public logradouro: string;

  @column()
  public numero: string;

  @column()
  public cidade: string;

  @column()
  public estado: string;

  @column()
  public tutorId: number;

  @column.dateTime({ autoCreate: true })
  public createdAt: DateTime;

  @column.dateTime({ autoCreate: true, autoUpdate: true })
  public updatedAt: DateTime;
}